src/SellOrder.sol

```diff
@@ -194,10 +201,21 @@ contract SellOrder {
         bool result1 = token.transfer(seller, orderStake);
         assert(result1);
 
-        // Transfer the payment to the seller
-        bool result2 = token.transfer(seller, offer.price);
+        uint256 toOrderBook = (offer.price * IOrderBook(orderBook).fee()) /
+            1000000;
```

Comment:
MAGIC NUMBER :(

Reply:
is `1000000` a const somewhere in the Open Zep lib?

Reply:
Can add a constant! 

We're defining a percentage as parts per 1,000,000, and this is the 1m!